cpp/src/groupby/hash/groupby.cu

```diff
@@ -158,6 +158,17 @@ class groupby_simple_aggregations_collector final
 
     return aggs;
   }
+
+  std::vector<std::unique_ptr<aggregation>> visit(
+    data_type, cudf::detail::correlation_aggregation const&) override
+  {
+    std::vector<std::unique_ptr<aggregation>> aggs;
+    aggs.push_back(make_sum_aggregation());
+    // COUNT_VALID
+    aggs.push_back(make_count_aggregation());
+
+    return aggs;
```

Comment:
If you want to be fancy. 
```suggestion
    return {make_sum_aggregation(), make_count_aggregation()};
```

Reply:
`std::initializer_list` only allows access to `const` elements. When `unique_ptr` is given in initializer list, `std::vector` tries to copy `unique_ptr`. copy constructor of `unique_ptr` is deleted.
So, this fails to compile.